docs/src/pages/style/typography/typography.md

```diff
@@ -11,15 +11,16 @@ A [typographic scale](https://material.io/design/typography/#type-scale) has a l
 
 ## General
 
-The *Roboto* font will **not** be automatically loaded by Material-UI.
+The _Roboto_ font will **not** be automatically loaded by Material-UI.
 The developer is responsible for loading all fonts used in their application.
 Roboto Font has a few easy ways to get started.
 
 ## Roboto Font CDN
 
 Shown below is a sample link markup used to load the Roboto font from a CDN.
+
 ```html
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
```

Comment:
That's HTML not JSX, I guess we can document both.

Reply:
Similar to the `_Roboto_` change, I'm not entire sure where this was introduced and I also couldn't reproduce the change seen here. Will revert.